### PR TITLE
[skia] Arm64 for skia on osx

### PR DESCRIPTION
**Pull request by bold84**
**Describe the pull request**

- #### What does your PR fix?  
  Allows skia to be built on Apple M1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  arm64-osx, no

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/skia/vcpkg.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "skia",
-  "version-date": "2020-05-18",
-  "port-version": 7,
+  "version-date": "2022-02-22",
```
Comment (JackBoosY) [inline ports/skia/vcpkg.json:3]: I think we shouldn't change the version-date.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/skia/vcpkg.json (hunk 1 above)
Comment (bold84) [reply ports/skia/vcpkg.json:3]: Shouldn't it be changed in baseline.json as well?

Comment (bold84) [reply ports/skia/vcpkg.json:3]: Ah, I get it now. But the port-version should be changed to 8?

Comment (JackBoosY) [reply ports/skia/vcpkg.json:3]: @bold84 Absolutly right,

Comment (bold84) [reply ports/skia/vcpkg.json]: @JackBoosY Done :-)

**Review by JackBoosY — APPROVED** [review]
LGTM if the tests passed.

**Pull request merged**